.rubocop.yml

```diff
@@ -1,3 +1,7 @@
+AllCops:
+  Exclude:
+    - '**/Rakefile'
```

Comment:
I'd rather not ignore all cops for these files, can we work through the issues together?